web/src/compose_ui.js

```diff
@@ -5,6 +5,7 @@ import autosize from "autosize";
 import $ from "jquery";
 import {insert, replace, set, wrapSelection} from "text-field-edit";
 
+import * as list_util from "./bulleted_numbered_list_util";
```

Comment:
We as a practice avoid importing modules with a renaming list this; it makes `git grep` much less effective for finding where something is used.

Reply:
Fixed in https://github.com/zulip/zulip/pull/22445#issuecomment-1726717296.